src/main/java/com/avispl/symphony/dal/communicator/aver/ptz/enums/ReplyStatus.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2021 AVI-SPL Inc. All Rights Reserved.
+ */
+package com.avispl.symphony.dal.communicator.aver.ptz.enums;
+
+/**
+ * This class is used to define reply status code
+ *
+ * @author Harry
+ * @version 1.0
+ * @since 1.0
+ */
+public enum ReplyStatus {
+	;
```

Comment:
remove this?

Reply:
ah, I will update the reply status enum now.